frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -347,6 +333,21 @@ const HostDetailsPage = ({
     }
   );
 
+  const canEditMdm = (() => {
+    const userHasPermission =
+      !!currentUser &&
+      !!host &&
+      (isGlobalAdmin ||
+        isGlobalMaintainer ||
+        permissionUtils.isTeamMaintainerOrTeamAdmin(currentUser, host.team_id));
```

Comment:
Unrelated to this PR, but I wonder what a simple way looks like to determine if the user can do/see something.